src/client/src/OpenFin/apps/Launcher/NlpSuggestions/executions/types.ts

```diff
@@ -0,0 +1,59 @@
+import { Direction } from "@/generated/TradingGateway"
+import { ExecutionTrade } from "@/services/executions"
+
+export enum NlpExecutionStatus {
```

Comment:
Need to have a deeper think about all these types and whether they do actually fit with the RFQ execution state

Reply:
I would agree .. there seems to be a mismatch between the data we are playing with and what we are declaring as our basic types - for the `rfqExecutionState$`, certainly, I haven't cross-checked how much of this already existed in TradeExecution, but I think the NLP return data is quite stable, isn't it? so there should be no need to be overly woolly in our typing